flashinfer/trace/templates/moe.py

```
@@ -91,8 +94,22 @@ def _fp8_moe_run_experts(
+        alpha = (
+            1.0
+            if gemm1_alpha is None
+            else gemm1_alpha[le].to(device=X2.device, dtype=torch.float32)
+        )
+        beta = (
+            0.0
+            if gemm1_beta is None
+            else gemm1_beta[le].to(device=X1.device, dtype=torch.float32)
+        )
+        silu_X2 = X2 * torch.sigmoid(alpha * X2)
+        O = (silu_X2 * (X1 + beta)).matmul(W2[le].t())
```

Ruff (0.15.15)

[error] Ambiguous variable name: `O`

(E741)